object_storage_api/routers/image.py

```diff
@@ -46,3 +46,27 @@ def create_image(
     logger.debug("Image upload file: %s", upload_file)
 
     return image_service.create(image_metadata, upload_file)
+
+
+@router.get(
+    path="",
+    summary="retrieve a list of images",
+    response_description="a list of images",
+    status_code=status.HTTP_200_OK,
+)
+def get_image_list(
```

Comment:
```suggestion
def get_images(
```
Consistent with IMS.